app/models/agents/twitter_user_agent.rb

```diff
@@ -57,12 +63,12 @@ def default_options
         'username' => 'tectonic',
         'include_retweets' => 'true',
         'exclude_replies' => 'false',
-        'expected_update_period_in_days' => '2'
+        'expected_update_period_in_days' => '2',
+        'choose_home_time_line' => 'false'
       }
     end
 
     def validate_options
-      errors.add(:base, "username is required") unless options['username'].present?
```

Comment:
How about making the `username` validation dependent on `choose_home_time_line` and ensure `choose_home_time_line` is set to `true` or `false`? (The latter needs a migration for existing `TwitterUserAgent`s)